tools/accuracy_checker/configs/efficientnet-b0-pytorch.yml

```diff
@@ -0,0 +1,96 @@
+models:
+  - name: efficientnet-b0-pytorch
+
+    launchers:
+      - framework: onnx_runtime
+        model: efficientnet-b0-pytorch/efficientnet-b0.onnx
+        adapytorcher: classification
+        cpu_extensions: AUTO
+        inputs:
+          - name: data
+            type: INPUT
+            shape: 1,3,224,224
+
+
+    datasets:
+      - name: imagenet_1000_classes
+        data_source: ImageNet
```

Comment:
data source should be removed. in light accuracy validation, it got from definitions.